modules/liveIntentIdSystem.js

```diff
@@ -66,7 +66,8 @@ function initializeLiveConnect(configParams) {
   const publisherId = configParams.publisherId || 'any';
   const identityResolutionConfig = {
     source: 'prebid',
-    publisherId: publisherId
+    publisherId: publisherId,
+    requestedAttributes: [ 'nonId' ].concat(configParams.extraRequestedAttributes || [ ])
```

Comment:
It looks like we have three proposals how to compute requestedAttributes:
- this one
- the one in the spec where one has to explicitly provide which attributes are requested: https://liveintent.atlassian.net/wiki/spaces/PL/pages/3394404382/Expose+UID2+Via+LiveIntent+s+Prebid+UserId+Module+WIP#Configuration
- the one in the ticket: https://liveintent.atlassian.net/browse/NAVY-484:
 
```
We also need a configuration option for publishers that defaults to true for each 
bid request - in future we could potentially do Ramp ID envelopes. So the json 
configuration could specify which ID they want to enrich in their web bid request
- some thing including name of id, true along with an optional fields that could 
pass anything we need to generate the identifiers they want us to enrich. 
If this config is set to false, we do not enrich bid request with the identifier.
```

I think either opting in or opting out should work.

Reply:
This is the same as the spec, with the one difference being that we always resolve nonId. That is important for our module to work correctly, so we should not allow users to disable it.

But good point regarding disabling some future default values. Will update this to operate on an object instead of a list

Reply:
🤦 misread `nonId` for `uid` :)

Reply:
>But good point regarding disabling some future default values. Will update this to operate on an object instead of a list

👍